guide/06-imagery-and-raster-analysis/orthomapping_guide_1.ipynb

```diff
@@ -0,0 +1,741 @@
+{
```

Comment:
<p>Add a line here saying you are defining a function that utilizes the <code>exifread</code> library to get location information out of the images. These coordinates would be utilized later in analysis.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/495/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>